index.html

```diff
@@ -0,0 +1,250 @@
+         <a href="/someurl" class="sec3rlink31">2015</a>
+       </li>
+    </ul>
+<p class="paragraph1">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
+
+<ul class="ad">
+<li class="canopli2">
+     <a href="/someurl" class="sec3rlink32">HTML</a>
+     <a href="/someurl" class="sec3rlink32">Ruby on rails</a>
+     <a href="/someurl" class="sec3rlink32">CSS</a>
+     <a href="/someurl" class="sec3rlink32">JAVASCRIPT</a>
+ </li>
+</ul>
+<button type="submit" class="bb sec3rbutton sec3rlink4"><span class="d">See Projects</span></button>
+  </div>
+  <div class="right">        
+    <img src="images/b.png" alt="">
+  </div>
+</section>
+
+<!-- Section 4 completed -->
+
+<!-- Section 5 started -->
+<section class="row4">
+  <div class="r4left abcl">
+    
+    <img src="images/c.png" alt="">
+
+  </div>
+
+  <div class="right">        
+    <h2 class="tonic">Facebook 360</h2>
+    <ul>
+      <li class="canopli">
+         <a href="/someurl" class="sec3rlink31">FACEBOOK</a>
+          <b class="sec3p">.</b>
+          <a href="/someurl" class="sec3rlink31">Full Stack Dev</a>
+           <b class="sec3p">.</b>
+         <a href="/someurl" class="sec3rlink31">2015</a>
+       </li>
+    </ul>
+<p class="paragraph1">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
+
+<ul class="ad">
+<li class="canopli2">
+     <a href="/someurl" class="sec3rlink32">HTML</a>
+     <a href="/someurl" class="sec3rlink32">Ruby on rails</a>
+     <a href="/someurl" class="sec3rlink32">CSS</a>
+     <a href="/someurl" class="sec3rlink32">JAVASCRIPT</a>
+ </li>
+</ul>
+<button type="submit" class="bb sec3rbutton sec3rlink4"><span class="d">See Projects</span></button>
+</div>
+</section>
+<!-- Section 5 closed -->
+
+<!-- Section 6 started -->
+<!-- Row five started-->
+<section class="row5">
+  <div class="r3left">
+    <h2 class="tonic">Uber Navigation</h2>
+    <ul>
+      <li class="canopli">
+         <a href="/someurl" class="sec3rlink31">Uber</a>
+          <b class="sec3p">.</b>
+          <a href="/someurl" class="sec3rlink31">Lead Developer</a>
+           <b class="sec3p">.</b>
+         <a href="/someurl" class="sec3rlink31">2018</a>
+       </li>
+    </ul>
+<p class="paragraph1">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+
+<ul class="ad">
+<li class="canopli2">
+     <a href="/someurl" class="sec3rlink32">HTML</a>
+     <a href="/someurl" class="sec3rlink32">Ruby on rails</a>
+     <a href="/someurl" class="sec3rlink32">CSS</a>
+     <a href="/someurl" class="sec3rlink32">JAVASCRIPT</a>
+ </li>
+</ul>
+<button type="submit" class="bb sec3rbutton sec3rlink4"><span class="d">See Projects</span></button>
+  </div>
+  <div class="right">        
+    <img src="images/d.png" alt="">
+  </div>
```

Comment:
Kindly make all the works resemble the Figma design for both desktop and mobile 🤗 as described previously.